src/client/contexts/RealtimeProvider.jsx

```diff
@@ -72,6 +72,22 @@ export function RealtimeProvider({ children }) {
       socket.on('meta changed', (payload) => {
         data.classroomMeta = payload;
       });
+
+      socket.on('question resolved', (payload) => {
+        const idx = data.messages.findIndex(x => x.id === payload.id);
+        if (idx >= 0) {
+          const messages = [...data.messages];
+          messages[idx] = payload;
+          data.messages = messages;
+          console.log(data.replyToMessageId, payload.id); // TODO: why is data.replyToMessageId null here?
```

Comment:
Need to investigate what's happening, but `data.replyToMessageId` is null here even though the "Reply to xxx's question" banner is still showing